core/iwasm/fast-jit/jit_regalloc.c

```diff
@@ -386,9 +383,8 @@ collect_distances(RegallocContext *rc, JitBasicBlock *basic_block)
 static JitReg
 offset_of_spill_slot(JitCompContext *cc, JitReg slot)
 {
-    /* TODO: check the spill */
-    return jit_cc_new_const_I32(
-        cc, /*cc->interp_frame_size + jit_cc_get_const_I32 (cc, slot) * 4*/ 0);
+    return jit_cc_new_const_I32(cc, cc->spill_cache_offset
```

Comment:
it may should longer than 16

Reply:
Same as above, the original code is:
```C
return jit_cc_new_const_I32(
        cc, cc->interp_frame_size + jit_cc_get_const_I32 (cc, slot) * 4);
```
I just replace `cc->interp_frame_size` to `cc->spill_cache_offset`

Reply:
I mean `spill_cache` is a `uint32` array with 16 elements only, it might not be long enough.